ddtrace/tracer.py

```diff
@@ -840,6 +841,40 @@ def shutdown(self, timeout=None):
             pass
         atexit.unregister(self._atexit)
 
+    def get_correlation_ids(self, tracer=None):
+        # type: (Optional[Tracer]) -> Tuple[Optional[int], Optional[int]]
+        """Retrieves the Correlation Identifiers for the current active ``Trace``.
+        This helper method can be achieved manually and should be considered
+        only a shortcut. The main reason is to abstract the current ``Tracer``
+        implementation so that these identifiers can be extracted either the
+        tracer is an OpenTracing tracer or a Datadog tracer.
+
+        OpenTracing users can still extract these values using the ``ScopeManager``
+        API, though this shortcut is a simple one-liner. The usage is:
+
+            trace_id, span_id = tracer.get_correlation_ids()
+
+        :returns: a tuple containing the trace_id and span_id
+        """
+        # Consideration: currently we don't have another way to "define" a
+        # GlobalTracer. In the case of OpenTracing, ``opentracing.tracer`` is exposed
+        # and we're doing the same here for ``ddtrace.tracer``. Because this helper
+        # must work also with OpenTracing, we should take the right used ``Tracer``.
+        # At the time of writing, it's enough to support our Datadog Tracer.
+
+        # If no tracer passed in, use self
+        if tracer is None:
+            tracer = self
+
+        # If tracer is disabled, skip
+        if not tracer.enabled:
+            return None, None
+
+        span = tracer.current_span()
+        if not span:
+            return None, None
+        return span.trace_id, span.span_id
```

Comment:
this is weird, but I wonder if we should return these as strings and/or return `0` instead of `None`?

that way it'll mimic the behavior of our logging code, which always uses strings and defaults to `"0"` instead of `None`

Reply:
so more like this?
```suggestion
        return "0", "0"
```

Reply:
If it does match what we do in `ddtrace.contrib.logging`, I'm all for making this change to return strings. But I think this will conflict with if we want to also maintain the same functionality as `helpers.get_correlation_ids()` as you mention below, WDYT? Or by "same functionality" do you mean regardless of format?

Reply:
Update: I've decided to add the same functionality (compatible) to our `ddtrace.opentracer.Tracer` wrapping, so that we can get rid of the `tracer=...` argument entirely and have this be a clean instance method. 

I'll make a note to change the output of this helper for a separate PR since that doesn't directly need to be addressed right now, especially if we want to maintain same functionality with the existing (deprecated) helper.